presentation/src/main/java/com/knocklock/presentation/ui/setting/credit/Developer.kt

```diff
@@ -0,0 +1,81 @@
+package com.knocklock.presentation.ui.setting.credit
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.knocklock.presentation.R
+
+data class Developer(
+    val name: String,
+    val profile: Painter,
+    val comment: String
+)
```

Comment:
```suggestion
data class Developer(
    @StringRes val nameRes: Int,
    @DrawableRes val profileRes: Int,
    @StringRes val commentRes: Int
)
```

Resource를 쓰는 시점에서 Ui Model을 좀더 원자적으로? 가져가도 좋을 것 같아요!

Reply:
리소스.. 끄적끄적 확인입니당